telecine/src/main/res/values/strings.xml

```diff
@@ -8,9 +8,9 @@
   <string name="tile_name">Launch Telecine</string>
   <string name="clear">Clear</string>
   <string name="record">Record</string>
-  <string name="countdown_one">1…</string>
-  <string name="countdown_two">2…</string>
-  <string name="countdown_three">3…</string>
+  <string name="countdown_one" translatable="false">1…</string>
+  <string name="countdown_two" translatable="false">2…</string>
+  <string name="countdown_three" translatable="false">3…</string>
```

Comment:
Hmm. Doesn't Arabic have it's own number glyphs? I swear some language doesn't use these glyphs...